packages/react-reconciler/src/ReactFiberWorkLoop.new.js

```diff
@@ -1972,6 +1972,7 @@ function commitRootImpl(root, renderPriorityLevel) {
     while (nextEffect !== null) {
       const nextNextEffect = nextEffect.nextEffect;
       nextEffect.nextEffect = null;
```

Comment:
Can this just be done inside `detachFiber` now?

Reply:
We do this for all effects, `detachFiber` only gets called on fibers that are flagged for deletion. So it's a different change.